x-pack/plugins/maps/public/classes/sources/kibana_tilemap_source/create_source_editor.js

```diff
@@ -13,10 +13,12 @@ import { i18n } from '@kbn/i18n';
 
 export function CreateSourceEditor({ onSourceConfigChange }) {
   const tilemap = getKibanaTileMap();
-
-  if (tilemap.url) {
-    onSourceConfigChange();
-  }
+  useEffect(() => {
+    if (tilemap.url) {
+      onSourceConfigChange();
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Passing an empty array to `useEffect` has the effect of only executing once on mount like `componentDidMount` for functional components. It's the generally accepted stand-in for `componentDidMount` but still triggers the linter